pubspec.yaml

```diff
@@ -28,29 +28,24 @@ dependencies:
   sockjs_client:
     git:
       url: git://github.com/Workiva/sockjs-dart-client.git
-      ref: 0.3.3
-  sockjs_client_wrapper: ^1.0.4
+      ref: 0.3.4
+  sockjs_client_wrapper: ^1.0.5
 
 dev_dependencies:
   browser: ^0.10.0+2
```

Comment:
need to remove browser and replace .dart with .dart.js in html files to load the compiled JS always